public/app/common/components/cc-form/cc-form.jsx

```diff
@@ -0,0 +1,36 @@
+var React = require('react');
+var styleMixin = require('mixins/style-mixin');
+var t = require('tcomb-form');
+var Form = t.form.Form;
+
+var options = {
+  templates: {
+    textbox: require('./ccTextInput.jsx')
+  }
+};
+
+var CCForm = React.createClass({
+  mixins: [
+    styleMixin(require('./style.scss'))
+  ],
+
+  onChange: function onChange (data) {
+    var validValue = this.refs.form.getValue();
```

Comment:
Thats API of the library http://gcanti.github.io/tcomb-form/guide/index.html#hello-form